src/edu/cs505/g5/OffState.java

```diff
@@ -0,0 +1,16 @@
+package edu.ccsu.cs505.g5;
+
+import java.util.Vector;
+
+/**
+ * This class extends RaspberriPiState and observer sends a notification (to the user) that RaspberryPi is going to switch off
+ * @author CS505-Group5 
+ */
+public class OffState extends RasberriPiState {
+
+    /**
+     * This observer sends a notification (to the user) that RaspberryPi is going to switch off
+     */
+    public Vector  myObserverNotification;

```

Comment:
You should not have any attribute types particularly on a public interface that are of a concrete type.  Use the Collections framework as was discussed in class and unless you explicitly need the Vector class because of multi-threading issues this isn't the right concrete class anyway.